packages/cli/src/commands/contract/compile.ts

```diff
@@ -95,3 +102,11 @@ export class CompileContract extends Command {
     }, "Writing config");
   }
 }
+
+// https://github.com/Supercolony-net/typechain-polkadot#usage-of-typechain-compiler
+export interface TypechainCompilerConfig {
```

Comment:
Does it need to be exported?

Reply:
Oh, no need to export. I'll update it, thank you.